### PR TITLE
Add errer handler and alerts component

### PR DESCRIPTION
**Pull request by DesignByOnyx**

### CODE REVIEW

File: public/lib/behaviors/error-handler.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import connect from 'can-connect';
+import hub from 'bitcentive/lib/hub';
```
Comment (justinbmeyer) [inline public/lib/behaviors/error-handler.js]: move to use can-event.

File: public/components/alerts/alerts.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import DefineMap from 'can-define/map/';
+import template from './alerts.stache';
+import './alerts.less';
+import hub from '../../lib/hub';
+import AlertItem from '../../models/alert';
+
+export const ViewModel = DefineMap.extend({
+  subscription: '*',
+  alerts: {
+    Value: AlertItem.List
```
Comment (justinbmeyer) [inline public/components/alerts/alerts.js]: use streams

File: public/lib/behaviors/error-handler.js (hunk 1 above)
Comment (DesignByOnyx) [inline public/lib/behaviors/error-handler.js]: https://github.com/donejs/bitcentive/issues/154

File: public/components/alerts/alerts.js (hunk 2 above)
Comment (DesignByOnyx) [inline public/components/alerts/alerts.js]: https://github.com/donejs/bitcentive/issues/149

**Comment by DesignByOnyx** [comment]
@justinbmeyer - the issues were created, and code comments were added in the appropriate places with links to their issues.

**Pull request merged**